docs/prometheus.md

```diff
@@ -0,0 +1,47 @@
+# Prometheus
+
+With Contour it is possible to get metrics from Envoy. In order for this to work properly,
+you must expose the Envoy admin socket and configure the Prometheus service discovery correctly.
+
+The admin socket can be made public by running
```

Comment:
replace with:
"Make the admin socket public:"
(the command makes it clear; you don't need the extra words :-) )